src/main/java/duke/task/ToDo.java

```diff
@@ -0,0 +1,37 @@
+package duke.task;
+import duke.exception.InvalidToDoException;
```

Comment:
Might be better to leave a space between the package and import statements since you did it for your other files for consistency reasons.